packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -209,55 +214,43 @@ export class ObjectForest extends SimpleDependee implements IEditableForest {
     }
 
     tryGet(
-        destination: ForestAnchor, cursorToMove: ITreeSubscriptionCursor, observer?: ObservingDependent | undefined,
+        destination: Anchor, cursorToMove: ITreeSubscriptionCursor, observer?: ObservingDependent,
     ): TreeNavigationResult {
-        assert(destination instanceof ObjectAnchor, 0x336 /* ObjectForest must only be given its own Anchors */);
-        assert(cursorToMove instanceof Cursor, 0x337 /* ObjectForest must only be given its own Cursor type */);
-        assert(cursorToMove.forest === this, 0x338 /* ObjectForest must only be given its own Cursor */);
-        const node = destination.find(this, observer);
-        if (node === undefined) {
+        const path = this.anchors.locate(destination);
+        if (path === undefined) {
             return TreeNavigationResult.NotFound;
         }
-        for (const [range, field] of this.roots) {
-            if (field.length !== 0) {
-                // TODO: it unclear if this should be allowed to modify cursor in the case
-                // it does not find the actual result.
-                cursorToMove.set(range, 0);
-
-                // Epically slow solution: search entire tree for node:
-                if (this.search(node, cursorToMove) === TreeNavigationResult.NotFound) {
-                    cursorToMove.clear();
-                } else {
-                    return TreeNavigationResult.Ok;
-                }
-            }
-        }
-
-        return TreeNavigationResult.NotFound;
+       this.getAtPath(path, cursorToMove, observer);
+        return TreeNavigationResult.Ok;
     }
 
-    private search(destination: JsonableTree, cursor: Cursor): TreeNavigationResult {
-        if (cursor.getNode() === destination) {
-            return TreeNavigationResult.Ok;
-        }
+    /**
+     * Set `cursorToMove` to location described by path.
+     * This is NOT a relative move: current position is discarded.
+     * Path must point to existing node.
+     */
+    getAtPath(destination: UpPath, cursorToMove: ITreeSubscriptionCursor, observer?: ObservingDependent): void {
```

Comment:
I would consider renaming this to something like `moveCursorTo`, `placeCursor`, or `setCursorPosition`, and similarly renaming the other `get` flavor(s).

Reply:
done